ERCS/erc-7802.md

```diff
@@ -0,0 +1,270 @@
+---
+eip: 7802
+title: Crosschain Token Interface
+description: Minimal token interface for cross-chain transfers
+author: skeletor (@skeletor-spaceman), parti (@0xParticle), joxes (@Joxess), ng (@0xng), agus duha (@agusduha), disco (@0xDiscotech), gotzen <gotzen@defi.sucks>, 0age <0age@uniswap.org>, Mark Tyneway <mark@oplabs.co>, Zain Bacchus <zain@oplabs.co>, Matt Solomon <msolomon@oplabs.co>, Maurelian <maurelian@protonmail.ch>, Blaine Malone (@blmalone)
+discussions-to: https://ethereum-magicians.org/t/erc-7802-crosschain-token-interface/21508
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-10-30
+requires: 165
+---
+
+## Abstract
+
+This standard introduces a minimal interface for tokens to communicate cross-chain. It allows bridges with mint and burn rights to send and relay token transfers with a standardized API.
```

Comment:
Not bad, but I'd like to see a bit more technical meat here. Could you sketch out _how_ your proposal operates, in addition to the description you already have.

Reply:
Added more technical details.